dashboard/test/models/user_test.rb

```diff
@@ -1504,6 +1504,12 @@ def complete_script_for_user(user, script, completed_date = Time.now)
     refute student.can_change_own_user_type?
   end
 
+  test 'cannot change own user type as a student in a section' do
+    section = create :section
+    student = create(:follower, section: section).student_user
```

Comment:
I believe the `:follower` factory will create a section if you don't provide one.
```rb
student = create(:follower).student_user
```
Or, it might be nice to extend our factories with an `:in_section` attribute for users
```rb
student = create :student, :in_section
```